src-lts/scripts/scr_catspeak_codegen/scr_catspeak_codegen.gml

```diff
@@ -1103,7 +1103,7 @@ function __catspeak_expr_index_set__() {
     var value_ = value();
     if (is_array(collection_)) {
         collection_[@ key_] = value_;
-    } else if (is_struct(collection_)) {
+    } else if (is_struct(collection_)) || (instance_exists(collection_)) {
```

Comment:
If you look above there is a `__catspeak_is_withable` which is capable of detecting a greater range of valid "indexable" values. Using `instance_exists` here may error with a difficult to understand error from the Catspeak users perspective.

Reply:
Actually I just realized `__catspeak_is_withable` also includes `is_struct`... Maybe I should be doing this with more restful eyes 😅